composer/airflow_1_samples/airflow_db_cleanup.py

```diff
@@ -358,10 +361,16 @@ def cleanup_function(**context):
 
         if ENABLE_DELETE:
             logging.info("Performing Delete...")
-            # using bulk delete
-            query.delete(synchronize_session=False)
-            session.commit()
-            logging.info("Finished Performing Delete")
+            if ("do_not_delete_by_dag_id" in context["params"] and 
+                    context["params"].get("do_not_delete_by_dag_id")):
```

Comment:
This can be simplified to `if context["params"].get("do_not_delete_by_dag_id"):` assuming that it retrieves the boolean value.